gym/lib/gym/runner.rb

```diff
@@ -155,14 +167,33 @@ def move_ipa
       ipa_path
     end
 
+    # copys framework from temp folder:
+
+    def copy_files_from_path(path)
+      UI.success "Exporting Files:"
+      framework_path = Dir[path]
+      framework_path.each do |f|
+        existing_file = File.join(File.expand_path(Gym.config[:output_directory]), File.basename(f))
+        # If the target file already exists in output directory
+        # we have to remove it first, otherwise cp_r fails even with remove_destination
+        # e.g.: there are symlinks in the .framework
+        if File.exist?(existing_file)
+          UI.important "Removing #{File.basename(f)} from output directory" if $verbose
+          FileUtils.rm_rf(existing_file)
+        end
+        FileUtils.cp_r(f, File.expand_path(Gym.config[:output_directory]), remove_destination: true)
+        UI.message "\t ▸ #{File.basename(f)}"
+      end
+      framework_path.join("\n")
```

Comment:
Return this explicitly if you need it, otherwise remove please (and get rid of the `framework_path` temporary)